doc/python/callbacks.rst

```diff
@@ -49,7 +49,7 @@ inside iteration loop.  You can also pass this callback function directly into X
         verbose_eval=False)
 
     dump = booster.get_dump(dump_format='json')
-    assert len(early_stop.stopping_history['Valid']['CustomErr']) == len(dump)
+    assert len(early_stop.stopping_history['Train']['CustomErr']) == len(dump)
```

Comment:
apologies for the mistake, I think we should modify the previous the `data_name` in line 39 to `data_name="Valid"` instead since it doesn't make sense to early stop with the training data.

Reply:
> apologies for the mistake, I think we should modify the previous the `data_name` in line 39 to `data_name="Valid"` instead since it doesn't make sense to early stop with the training data.

yes, you are right! I have changed it again, please review, thanks!